js/main.js

```diff
@@ -457,10 +462,15 @@ jQuery(document).on('ready', function() {
                           <div class="tg-contentbox">
                             <div class="tg-eventhead">
                             <div class="tg-leftarea">
-                              <time datetime="2017-12-12">`+ nrow[0] +`</time>
-                              <div class="tg-title">
-                                <h2>`+ nrow[1] +`</h2>
-                              </div>
+                              <time datetime="2017-12-12">`+ nrow[0] + `</time>
+                              <div class="tg-title talk_id "  data-id='talk` + nrow[4] + `' >
+                              <h2>` + nrow[1] + `</h2>
+                            </div>
+                            
+                            <div class="tg-description" id='desc` + nrow[4] + `'>
```

Comment:
@CuriousLearner The div with ```tg-description``` loads talk description from ```data/tracks.json.```. By default, it's hidden (mentioned in ```css/main.css```)

Reply:
Yeah, I've seen that. What I meant was that `tracks.json` wasn't updated with the details.

Reply:
Oh. Okay.

Since there's a separate issue for it - #132, therefore I didn't.

Reply:
We'll have a 'cfp' key also when #137 will be merged, could you add also show 'CFP Proposal' with the value of that key. It'll take the user to that particular talk's CFP proposal. Also is the 'cfp' is empty make sure to not add the entire div with 'CFP Proposal'.